javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -575,7 +576,7 @@ public void register(AstObserver observer, ObserverRegistrationMode mode) {
         if (mode == null) {
             throw new IllegalArgumentException("Mode should be not null");
         }
-        switch (mode) {
+        switch(mode) {
```

Comment:
Do we have a style guideline? I tend to put spaces before parenthesis, I am happy to change my habits but I think we should write a common guideline somewhere (perhaps it exists and I am just missing it)